pgmq-extension/sql/pgmq--1.4.1--1.4.2.sql

```diff
@@ -187,6 +204,70 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
+
```

Comment:
I don't see these next two functions in the `pgmq.sql` diff; should they be?

Reply:
Here's the [drop_queue](https://github.com/tembo-io/pgmq/pull/300/files#r1720336769) change